From: John Paul Adrian Glaubitz <glaubitz@physik.fu-berlin.de>
Date: Sun, 14 Aug 2016 21:41:40 +0200
Subject: Fix declaration of static members in class Plotter.

---
 include/kc/plotter.h | 16 ++++++++--------
 1 file changed, 8 insertions(+), 8 deletions(-)

diff --git a/include/kc/plotter.h b/include/kc/plotter.h
index 552ef64..c4333df 100644
--- a/include/kc/plotter.h
+++ b/include/kc/plotter.h
@@ -28,28 +28,28 @@ class Plotter
 {
 private:
   /** paper width in mm (A4) */
-  static const double WIDTH_MM = 210.0;
+  static constexpr double WIDTH_MM = 210.0;
 
   /** paper height in mm (A4) */
-  static const double HEIGHT_MM = 297.0;
+  static constexpr double HEIGHT_MM = 297.0;
 
   /** plotting area width in mm */
-  static const double PLOTTING_AREA_WIDTH_MM = 188.0;
+  static constexpr double PLOTTING_AREA_WIDTH_MM = 188.0;
 
   /** plotting area height in mm */
-  static const double PLOTTING_AREA_HEIGHT_MM = 255.0;
+  static constexpr double PLOTTING_AREA_HEIGHT_MM = 255.0;
 
   /** scaling factor for the buffer image surface */
-  static const double BUFFER_SURFACE_SCALE = 10.0;
+  static constexpr double BUFFER_SURFACE_SCALE = 10.0;
 
   /** scaling factor for the onscreen image surface */
-  static const double ONSCREEN_SURFACE_SCALE = 3.0;
+  static constexpr double ONSCREEN_SURFACE_SCALE = 3.0;
 
   /** plotter step width in cm */
-  static const double STEP_WIDTH = 0.1;
+  static constexpr double STEP_WIDTH = 0.1;
 
   /** convert mm to points per inch (for pdf output) */
-  static const double MM_TO_INCH = 72.0 / 25.4;
+  static constexpr double MM_TO_INCH = 72.0 / 25.4;
 
 private:
   /** current coordianates of the plotting pen */
